README.rst

```diff
@@ -55,6 +55,9 @@ Enable tab completion with ``source az.completion.sh``.
 
 Some prerequisites may be required. See our `Preview Install Guide <https://github.com/Azure/azure-cli/blob/master/doc/preview_install_guide.md>`__.
 
+If you run into an ``AttributeError: 'X509' object has no attribute '_x509'`` error, downgrade your version of the requests library from 2.12.1 to 2.11.1.
```

Comment:
Does it make it sense to put it under a new section called `workaround` to make it more visible, since almost every installation will hit this issue